package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "commitplease",
-  "version": "2.3.1",
+  "version": "2.4.0",
```

Comment:
Ah, I understand now. My mistake when I was tagging and releasing 2.4.0: I pushed the end result to my origin, which is all3fox/commitplease. Once this branch is merged, I will remove all3fox/commitplease and make this repo my origin.

I will push two commits (the `2.4.0` tag and some tests) directly to master here (my upstream), that will fix the confusion.